benchmark/misc/freelist.js

```diff
@@ -0,0 +1,38 @@
+'use strict';
+
+var common = require('../common.js');
+var FreeList = require('../../lib/internal/freelist').FreeList;
```

Comment:
This should be `require('internal/freelist')` with a note at the top that `--expose-internals` is needed to run the benchmark. Until benchmarks get special "Flags" code comments that get parsed like tests have, flags will need to be explicitly specified on the command-line.